### PR TITLE
Changed page for error message 500

### LINKED ISSUE #5551: Add a link to report a bug on github to the 500 error page
If a user comes across a 500 error, it would be helpful to both us and them if they could easily file an issue on github with the relevant information (incl. steps to reproduce and sentry information) from that 500 page.

I think @EnTeQuAk has some more insights here.

### PR DESCRIPTION
**Pull request by obviyus**
Fixes #5551

### CODE REVIEW

**Comment by obviyus** [comment]
@diox I hope this is alright.

File: src/olympia/amo/templates/amo/500.html [hunk 1]
```diff
@@ -4,15 +4,18 @@
...
+<h1>Oops! We had an error.</h1>
 
 <p>We'll get to fixing that soon.</p>
 <p>
   You can try refreshing the page, or head back to the
-  <a href="{{ home }}">Add-ons homepage</a>.
+  <a href="{{ home }}">Add-ons homepage</a>. You can also help us fix this by
+  <a href="https://github.com/mozilla/addons/issues/new">filing an issue</a> with the
+  details of how you got to this page. Make sure to include this error reference
+  number: {{ ref }}.
```
Comment (diox) [inline src/olympia/amo/templates/amo/500.html]: Remove that last sentence as we don't support error references yet.

Comment (obviyus) [reply src/olympia/amo/templates/amo/500.html]: I've made the change.

**Comment by caitmuenster** [comment]
Thanks for the patch, @obviyus! ⭐️ Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Would you be interested in creating an account on mozillians.org? I'd be happy to vouch for you!

**Comment by obviyus** [comment]
Thanks for the addition @caitmuenster! I'd be glad to make an account.

**Comment by obviyus** [comment]
Here's my account link: https://mozillians.org/en-US/u/obviyus/

**Comment by caitmuenster** [comment]
Awesome! You're vouched. Welcome onboard! We look forward to seeing you around.

**Pull request merged**